src/test/ui/traits/negative-impls/pin-unsound-issue-66544-clone.stderr

```diff
@@ -0,0 +1,13 @@
+error[E0119]: conflicting implementations of trait `std::clone::Clone` for type `&mut MyType<'_>`:
+  --> $DIR/pin-unsound-issue-66544-clone.rs:7:1
+   |
+LL | impl<'a> Clone for &'a mut MyType<'a> {
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = note: conflicting implementation in crate `core`:
+           - impl<T> std::clone::Clone for &mut T
```

Comment:
This diagnostic is misleading, as it implies that `&'a mut MyType<'a>` _does_ implement `Clone`, rather than opting out.

Reply:
Yeah, good call. Should be fixable fairly easily I imagine.